### PR TITLE
ran tests, corrected a few things started on help doc

### PR DESCRIPTION
**Pull request by Marleyman876**

### CODE REVIEW

**Review by mhendricks96 — APPROVED** [review]

File: daring_fireball.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def main():
     
     os.system("clear")
     print(welcome("> prompt"))
-    print("\Welcome! We help you by checking availability of your items at the following stores: Best Buy, Newegg, Footlocker")
```
Comment (kevinhenry) [inline daring_fireball.py:50]: You shoul dbe able to leave the slash in place.

**Review by danieldills — APPROVED** [review]

File: daring_fireball.py [hunk 2]
```diff
@@ -71,19 +71,19 @@ def main():
         fcs.fc_site_search('http://www.flightclub.com/', search_string)
     elif choice == '3' :
         import footlocker_scraper as fot
-        fot.flocker_site_search('http://www.footlocker.com/', search_string)
+        fot.flocker_site_search('https://www.footlocker.de/', search_string)
```
Comment (kevinhenry) [inline daring_fireball.py:74]: This should be left as .com

**Review by kevinhenry — APPROVED** [review]
Looks good overall. Just those two minors.

File: daring_fireball.py (hunk 1 above)
Comment (Marleyman876) [reply daring_fireball.py:50]: it prints \welcome instead of Welcome bud.

Comment (Marleyman876) [reply daring_fireball.py:50]: slash has to go sorry 😄

**Pull request merged**